src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -65,6 +66,16 @@ const NavigationSidebarContent: FC<NavigationSidebarProps> = ({
     <NavigationSidebarMainMenu mainMenuItems={mainMenuItems} />
   ) : null;
 
+  const handleArrowClick = () => {
+    if (isThirdLevelMenuOpen) {
+      toggleOffThirdLevelMenu();
+      setShouldShowThirdLevel(false);
+    } else {
+      toggleOnThirdLevelMenu();
+      setShouldShowThirdLevel(true);
```

Comment:
What's the difference between `toggleOffThirdLevelMenu / toggleOnThirdLevelMenu` and `setShouldShowThirdLevel`? It seems like they may accomplishing the same functionality, at least based on the naming

Reply:
I keep information in this variable whether level 3 should be open or closed after closing the entire sidebar. Because the `toggleOffThirdLevelMenu` function is always used when closing the sidebar

Reply:
I agree with Armando. "toggle" means setting something from off to on or the other way round. toggleOff and toggleOn are a bit confusing to what exactly they're doing.

Reply:
This naming is used in about 40 files so maybe we'll leave it for the future?

Reply:
Sure, I don't have a problem with that.